src/OpenTracing/IScope.cs

```diff
@@ -0,0 +1,95 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AbstractTag.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OpenTracing
+{
+    using System;
```

Comment:
I want to use coding guidelines that follow recent OSS practices by Microsoft and I think .NET core and ASP.NET Core are the go-to for this. Following this, usings should stay outside of the namespace at the beginning of the file.

Reply:
Silly Microsoft, can't even agree consistently in what to suggest to the public, our internal stuff says the opposite (there's some weird dependency resolving/performance thingie with where they're placed that makes inside better for some cases, but historically people prefer outside).

Not blocking so I'll update, thanks!

Reply:
resolved